tests/nested/manual/muinstaller-core/task.yaml

```diff
@@ -75,7 +75,11 @@ execute: |
   # Retrieve kernel
   snap download --basename=pc-kernel --channel="$version/edge" pc-kernel
   # Build kernel with initramfs with the compiled snap-bootstrap
-  uc20_build_initramfs_kernel_snap "$PWD/pc-kernel.snap" "$NESTED_ASSETS_DIR"
+  if os.query is-ubuntu-ge 24.04; then
+    uc24_build_initramfs_kernel_snap "$PWD/pc-kernel.snap" "$NESTED_ASSETS_DIR"
```

Comment:
possibly worth putting this conditional inside of a generic `build_initramfs_kernel_snap` function?